commitlint/plugins.test.ts

```diff
@@ -476,6 +476,17 @@ test("footer-references-existence4", () => {
     expect(footerReferenceExistence4.status).toBe(0);
 });
 
+test("footer-references-with-EOL", () => {
+    let commmitMsgwithEOLFooter =
+        "foo: this is only a title" +
+        "\n\n" +
+        "Bla bla blah[1]" +
```

Comment:
@parhamsaremi this commit msg will make the paragraph rule fail

Reply:
I added a . after Bla bla.